assets/css/main.scss

```diff
@@ -54,3 +54,23 @@ p.sponsor-quote {
         max-width: 100%;
         box-shadow: 0 0 0 1px rgba(0,0,0,0.2);
 }
+
+a.site-title {
+    width: 30%;
+}
+
+.home .content {
+    margin-top: 2em;
+
+    > .logo {
+        width: 30%;
```

Comment:
If you put `min-width: 10em;` below the existing `width` line, the graphic will be shown a bit larger on narrow mobiles but will be the same size on the widest displays.

Reply:
Done